### PR TITLE
Simulation structure interfacing docs update

### PR DESCRIPTION
**Pull request by TorkelE**

### CODE REVIEW

**Comment by TorkelE** [comment]
Not sure what to do about the `setu` section. It has the same issues for problems as normal indexing update, but is potentially relevant if someone wishes to simulate a problem for a very large number of conditions. Thinking of just sticking a warning there saying that one should be careful when applying it in certain situations. Not sure what more situations there is right now than eliminated conservative laws though.

File: docs/src/model_simulation/simulation_structure_interfacing.md [hunk 1]
```diff
@@ -197,4 +183,4 @@ oprob[two_state_model.X1 + two_state_model.X2]
 This can be used to form symbolic expressions using model quantities when a model has been created using the DSL (as an alternative to @unpack). Alternatively, [creating an observable](@ref dsl_advanced_options_observables), and then interface using its `Symbol` representation, is also possible.
 
 !!! warn
```
Comment (isaacsas) [inline docs/src/model_simulation/simulation_structure_interfacing.md:185]:
```suggestion
!!! warning
```

File: docs/src/model_simulation/simulation_structure_interfacing.md [hunk 2]
```diff
@@ -197,4 +183,4 @@ oprob[two_state_model.X1 + two_state_model.X2]
...
-    With interfacing with a simulating structure using symbolic variables stored in a `ReactionSystem` model, ensure that the model is complete.
\ No newline at end of file
+    With interfacing with a simulating structure using symbolic variables stored in a `ReactionSystem` model, ensure that the model is complete.
```
Comment (isaacsas) [inline docs/src/model_simulation/simulation_structure_interfacing.md:186]:
```suggestion
    When accessing a simulation structure using symbolic variables from a `ReactionSystem` model, such as `rn.A` for `rn` a `ReactionSystem` and `A` a species within it, ensure that the model is complete.
```

**Review by isaacsas — CHANGES_REQUESTED** [review]
A couple comments, but then fine to merge.

**Pull request merged**